beacons-data/export_legacy_beacons.sql

```diff
@@ -1,18 +1,26 @@
 SELECT 
-	hex_id, 
-	UPPER(data -> 'beacon' ->> 'departRefId')AS dept_ref, 
-	UPPER(beacon_status) AS beacon_status,
-	TO_CHAR(TO_TIMESTAMP(data -> 'beacon' ->> 'firstRegistrationDate', 'YYYY-MM-DDXHH24:MI:SS.MS'), 'dd/mm/yyyy') AS created_date, 
-	TO_CHAR(last_modified_date :: DATE, 'dd/mm/yyyy') AS last_modified_date,
-	UPPER(data -> 'owner' ->> 'ownerName')AS owner_full_name,
-	UPPER(data -> 'owner' ->> 'email')AS owner_email,
-	UPPER(data -> 'owner' ->> 'address1')AS owner_address_line1,
-	UPPER(data -> 'owner' ->> 'address2')AS owner_address_line2,
-	UPPER(data -> 'owner' ->> 'address3')AS owner_address_line3,
-	UPPER(data -> 'owner' ->> 'address4')AS owner_address_line4,
-	UPPER(data -> 'owner' ->> 'postCode')AS owner_postcode,
-	UPPER(data -> 'owner' ->> 'county')AS owner_county,
-	UPPER(data -> 'owner' ->> 'townOrCity')AS owner_town_or_city,
-	UPPER(data -> 'owner' ->> 'country')AS owner_country
+	legacy_beacon.hex_id, 
+	UPPER(legacy_beacon.data -> 'beacon' ->> 'departRefId')AS dept_ref, 
+	CASE 
+		WHEN legacy_beacon_claim_event.claim_event_type IS NOT NULL AND legacy_beacon_claim_event.claim_event_type = 'claim' 
```

Comment:
Might be worth having this as two stages, so the null check is only done once:

WHEN legacy_beacon_claim_event.claim_event_type IS NOT NULL
    CASE
          WHEN legacy_beacon_claim_event.claim_event_type = 'claim' THEN 'DELETED (CLAIMED)' 
          WHEN legacy_beacon_claim_event.claim_event_type = 'reject' THEN 'DELETED (REJECTED)'
    END
    ELSE legacy_beacon.beacon_status

Reply:
Alternatively something like this might work? 
WHEN legacy_beacon_claim_event.claim_event_type = 'claim' THEN 'DELETED (CLAIMED)'
WHEN legacy_beacon_claim_event.claim_event_type = 'reject' THEN 'DELETED (REJECTED)'
WHEN legacy_beacon_claim_event.claim_event_type IS NULL THEN legacy_beacon.beacon_status